.github/workflows/tests.yml

```
@@ -24,10 +24,10 @@ on:
 jobs:
   python-tests:
     strategy:
-      max-parallel: 9
+      max-parallel: 3
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
        os: [ubuntu-latest, macos-latest]
```
I think we could remove windows from our CI tests.